2025-08-05 08:58:53,289 - ERROR - [router_async_vector_api] - 查询失败: ConnectionTimeout caused by - TimeoutError()
2025-08-05 10:11:17,658 - ERROR - [router_async_vector_api] - 批量查询失败: name 'query_by_uuids' is not defined
2025-08-05 13:47:38,370 - ERROR - [router_async_vector_api] - 批量查询失败: unhashable type: 'list'
